definitions/mobile-application/golden_metrics.yml

```diff
@@ -1,3 +1,17 @@
+appLaunchCount:
```

Comment:
could we add the correct units to the GMs? the ones used in the SM definition file seem to be correct